modules/ossm-about-adding-projects-using-smm.adoc

```diff
@@ -5,6 +5,8 @@
 [id="ossm-about-adding-projects-using-smm_{context}"]
 = About adding projects using the ServiceMeshMember resource
 
-A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMemberRoll` resource. Then, the services that reside in the project are made available to the mesh. 
+A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMember` resource. Then, the services that reside in the project are made available to the mesh. 
```

Comment:
Please revert this change. The `status.members` is in the `ServiceMeshMemberRoll` resource.

Reply:
Thanks for catching this Marko. The change was made and appears in the preview build.